src/lib/ubirch/ubirch_sim.py

```diff
@@ -274,10 +268,10 @@ def _execute(self, cmd: str) -> (bytes, str):
             cmd = cmd[0] + channel_char + cmd[2:]
 
         at_cmd = 'AT+CSIM={},"{}"'.format(len(cmd), cmd.upper())
-        result = self._send_at_cmd(at_cmd)
+        result = self.lte.send_at_cmd(at_cmd)
 
-        if result[-1] == 'OK':
-            response = result[0][7:].split(',')[1]
+        if result is not None:
```

Comment:
So if `result is None` the Exception below is not subscriptable! Compare Waldi's logs:
```
(2021.01.30 08:03:16) Traceback (most recent call last):
  File "main.py", line 245, in <module>
  File "/flash/lib/helpers.py", line 54, in send_backend_data
  File "/flash/lib/ubirch/ubirch_sim.py", line 181, in deinit
  File "/flash/lib/ubirch/ubirch_sim.py", line 178, in deinit
  File "/flash/lib/ubirch/ubirch_sim.py", line 242, in _close_channel
  File "/flash/lib/ubirch/ubirch_sim.py", line 281, in _execute
TypeError: 'NoneType' object isn't subscriptable
```